modules/store-devtools/src/reducer.ts

```diff
@@ -123,12 +123,12 @@ function recomputeStates(
     const entry: ComputedState = shouldSkip
       ? previousEntry
       : computeNextEntry(
-        reducer,
```

Comment:
Revert these changes

Reply:
@brandonroberts 
I created a pull request to fix the indentations.
https://github.com/ngrx/platform/pull/1301